### DISCUSSION TITLE
Unable to delete revision of first Issue comment (the OP)

### DISCUSSION THREAD

**Discussion by YoshiRulz** [community/community#28816] [New to GitHub]
I have edit perms for [this repo](https://github.com/TASEmulators/BizHawk). The OP of [this Issue](https://github.com/TASEmulators/BizHawk/issues/3231) contained an unacceptable link which, though edited out, remains in the history. I cannot delete the original revision, nor does the "The most recent revision cannot be deleted." message appear for the current revision, as there is no "Options" dropdown to open in the first place. The dropdown is shown for a comment under the Issue.

**Comment by airtower-luna** [comment]
That sounds like something to [contact GitHub support](https://support.github.com/request) about, if anyone can override that limitation it's them.
